lib/msf/core/exploit/socket_server.rb

```diff
@@ -18,7 +18,7 @@ def initialize(info = {})
 
     register_options(
       [
-        OptAddress.new('SRVHOST', [ true, "The local host to listen on. This must be an address on the local machine or 0.0.0.0", '0.0.0.0' ]),
+        OptAddressLocal.new('SRVHOST', [ true, 'The local host or network interface to listen on. This must be an address on the local machine or 0.0.0.0.', '0.0.0.0' ]),
```

Comment:
```suggestion
        OptAddressLocal.new('SRVHOST', [ true, 'The local host or network interface to listen on. This must be an address on the local machine or 0.0.0.0 to listen on all addresses.', '0.0.0.0' ]),
```

@cnotin: This is a good compromise, and it's more descriptive. Not everyone knows what all-zeroes means.

Reply:
Good idea!